common/changes/@rushstack/heft-localization-typings-plugin/user-ianc-json-schema-typings-plugin_2025-07-07-22-10.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-localization-typings-plugin",
+      "comment": "Ensure the `srcFolder`, `trimmedJsonOutputFolders`, `generatedTsFolder`, and `secondaryGeneratedTsFolders` options don't contain backslashes.",
```

Comment:
If this was your intent, the pattern should be `^[^\\\\]+$`, or using `*` instead of `+` if empty string is valid.